### ISSUE TITLE
Cat out a sample answers.conf file during install

### ISSUE THREAD

**Issue by aweiteka** [projectatomic/atomicapp#26]
I would like to be able to...

1. install without providing an answerfile
1. have a sample `answerfile.conf.sample` file appear in my working directory
1. edit and save sample as `answerfile.conf`
1. run

**Comment by vpavlin** [comment]
Makes sense..what should be the content?

```
[general]
provider: $default_provider
```

`$default_provider` is currently kubernetes

**Comment by goern** [comment]
+1

**Comment by aweiteka** [comment]
Example. Given this Nulecule file...
```
...
graph:
  db-app:
    source: docker://aweiteka/db:app
  frontend-app:
    params:
      db_pass:
        description: Database password
      public_ip:
        description: Public IP address
...
```

I would expect `install` to create an `answers.conf` file with these contents:
```
[general]
provider=kubernetes

[frontend-app]
db_pass=
wp_public_ip=

[db-app]
db_pass=
```

**Comment by aweiteka** [comment]
Related to larger discussion here #27

**Comment by vpavlin** [comment]
Yes, this makes a lot of sense

**Comment by aweiteka** [comment]
Resolved by #34
